EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -92,6 +92,34 @@ public struct MediaPlayerView: View {
             .tint(color)
             .wrapInButton(action: { viewModel.onTapAction?(control) })
     }
+
+    private func timeFormat(_ timeInSeconds: Float64) -> String {
+        let timeInSeconds = Int(timeInSeconds)
+        let hours:Int = Int(timeInSeconds / 3600)
+        let minutes:Int = Int(timeInSeconds % 3600 / 60)
+        let seconds:Int = Int((timeInSeconds % 3600) % 60)
+
+        if Int(viewModel.playerDuration / 3600) > 0 {
+            return String(format: "%i:%02i:%02i", hours, minutes, seconds)
+        } else {
+            return String(format: "%02i:%02i", minutes, seconds)
+        }
```

Comment:
Move the condition `Int(viewModel.playerDuration / 3600) > 0` to a function for better understanding of what the condition means